### DISCUSSION TITLE
Backup strategy

### DISCUSSION THREAD

**Discussion by VoidAndAny** [tomMoulard/make-my-server#30] [Q&A]
Thanks for your repository, it is very inspiring.
How do you manage your backup ?

Yoann

**Comment by tomMoulard** [comment]
Hello there @VoidAndAny,

You are touching a very important subject!

For now, I'm simply having a cronjob that runs `rsync` on a regular basis. `rsync` synchronize all my data with two off-site backups.

As my data is mostly junk (movies, TV shows and music), I do not need others backup strategies.

But if other users have a different opinion, feel free to discuss this further here or on Discord.

**Reply by VoidAndAny** [reply]
I used to use restic so I give resticker image a try but for now it's a pain (PRE_COMMAND fail, need to mount all volume to backup in resticker)
I think it will be more easy to execute restic on host

Have you a dedicated Discord ?

**Reply by tomMoulard** [reply]
I've never tried restic :/

> Have you a dedicated Discord ?

For sure, it's there : https://discord.gg/zQV6m9Jk6Z
